emhttp/plugins/dynamix.plugin.manager/include/UnraidCheck.php

```diff
@@ -115,7 +115,14 @@ function _($text) {return $text;}
 
         $params  = [];
         $params['branch']          = plugin('category', self::PLG_PATH, 'stable');
-        $params['current_version'] = plugin('version', self::PLG_PATH) ?: _var($var,'version');
+        // Get current version from patches.json if it exists, otherwise fall back to plugin version or var.ini
+        $patcherVersion = null;
+        if (file_exists('/tmp/Patcher/patches.json')) {
```

Comment:
flagging to ensure this updates

Reply:
[`14e0fed` (#2099)](https://github.com/unraid/webgui/pull/2099/commits/14e0fed12a6cbdda0e5cd9c9cf7d37015085820e)